tests/browsercontext-cookies.spec.ts

```diff
@@ -137,14 +137,17 @@ it('should get cookies from multiple urls', async ({context}) => {
     url: 'https://foo.com',
     name: 'doggo',
     value: 'woofs',
+    sameSite: 'Lax',
```

Comment:
Do I understand correctly that if we add sameSite: None here, we'll get None at line 161? Let's mix and match different SameSite values to see that all of them work?

Reply:
done